.storybook/components/DesignTokens/Tier2/TypographyUsage.jsx

```diff
@@ -1,89 +1,40 @@
+import { at, capitalize, forEach } from 'lodash';
+
 import React, { Component } from 'react';
 import { Grid, Section } from '../../../../src';
+import usages from '../../../../src/design-tokens/tier-2-usage/typography.json';
+import flatten from '../../../util/flattenToken';
 import { TokenSpecimen } from '../../TokenSpecimen/TokenSpecimen';
-import { PRESET_SIZE_MAP } from '../Tier1/TypographyPresets';
-import styles from './TypographyUsage.module.css';
 
 export class Tier2TypographyUsage extends Component {
   render() {
-    const renderTypeToken = (size, preset, legacySize) => {
-      // remove variant suffix
-      const normalizedPreset = preset.split('-')[0];
-      const { fontSize, lineHeight } = PRESET_SIZE_MAP[normalizedPreset];
-
-      const name =
-        `eds-theme-typography-${size}` + (legacySize ? `/ ${legacySize}` : '');
-
-      return (
-        <TokenSpecimen
-          comment={`Ref: preset-${preset}`}
-          name={name}
-          specimenClassName={styles[`typography-usage--${size}`]}
-          value={`${fontSize} / ${lineHeight}`}
-          variant="typography-body"
-        />
+    const values = {};
+    forEach(at(usages, 'eds.theme.typography')[0], (_, key) => {
```

Comment:
Given the values in the typography file at the path `eds.theme.typography`, create a flat map of each one such that stuff like `{ a: { b: 'c' } }` becomes `{ a-b: 'c' }` at any nesting level.